src/Controls/src/Core/PointerPlatformEventArgs.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+#pragma warning disable RS0016 // Add public types and members to the declared API
+public class PointerPlatformEventArgs
```

Comment:
More of a nit, but the name feels weird to say... Both right and wrong.

Maybe `PlatformPointerEventArgs`?

Reply:
Updated!